### PR TITLE
Decrypt tool

### PR DESCRIPTION
**Pull request by leviyehonatan**
A new tool to allow users to read messages sent from vurnability submitters,

This tool is a vault for pgp keys that is saved locally on the browser's store and is unlocked using a single password to allow to collect multiple keys for the ease of use of committee members to audit submitted bugs

### CODE REVIEW

File: src/components/CommitteeTools/CommitteTools.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    const createVault = () => {
+        if (passwordRef.current!.value !== passwordConfirmRef.current!.value) {
+            setError("Passwords mismatch")
+            return
+        }
+        onCreateVault(passwordRef.current!.value)
+    }
+
+    return (
+        <div>
```
Comment (roienatan) [inline src/components/CommitteeTools/CommitteTools.tsx:109]: The `<div>` is unnecessary here

**Review by roienatan — CHANGES_REQUESTED** [review]

**Review by roienatan — CHANGES_REQUESTED** [review]
As we implemented the support of i18next, we need for any new development to use it, also here.

**Review by leviyehonatan — COMMENTED** [review]
looks good

**Review by leviyehonatan — COMMENTED** [review]
looks good to me

**Review by roienatan — APPROVED** [review]

**Pull request merged**